.golanggci.yml

```diff
@@ -0,0 +1,37 @@
+run:
+  concurrency: 4
+  timeout: 1m
+  issues-exit-code: 1
+  tests: true
+  build-tags:
+    - mytag
+  modules-download-mode: readonly
+  allow-parallel-runners: false
+
+output:
+  format: colored-line-number
+  print-issued-lines: true
+  print-linter-name: true
+  uniq-by-line: true
+  path-prefix: ""
+  sort-results: false
+
+linters-settings:
+  cyclop:
+    max-complexity: 10
+    package-average: 0.0
+    skip-tests: false
+
+linters:
+  disable-all: true
+  enable:
+    - megacheck
+    - govet
+  enable-all: true
+  disable:
+    - maligned
+    - prealloc
+  presets:
```

Comment:
I'd propose do not use presets. Just use linter as is with overridden max-complexity param